src/lambda/explorer/defaultRegionNode.ts

```diff
@@ -70,7 +75,7 @@ export class RegionNodeCollection {
             this.regionNodes,
             regionMap.keys(),
             key => this.regionNodes.get(key)!.update(regionMap.get(key)!),
-            key => new DefaultRegionNode(regionMap.get(key)!)
+            key => new DefaultRegionNode(regionMap.get(key)!, this.getExtensionAbsolutePath)
```

Comment:
Unbound instance method, see above.

Reply:
There are several other instances of this, please address throughout the changeset.